worked-solutions/01_penguin_classification_solutions.ipynb

```diff
@@ -509,7 +514,7 @@
     "# Create an optimiser and give it the model's parameters.\n",
     "from torch.optim import Adam\n",
     "\n",
-    "optimiser = Adam(model.parameters())"
+    "optimiser = Adam(model.parameters(), lr=1e-4)"
```

Comment:
IIRC the choice of a learning rate is discussed in exercise 2, we left it as the default here so as not to over-complicate/increase the knowledge burden on exercise 1.

Reply:
1e-4 is super common with Adam, but so long as the model works it is fine to leave it as it was.